### DISCUSSION TITLE
SD card method does not work

### DISCUSSION THREAD

**Discussion by quantumfrost** [e27-camera-hack/E27-Camera-Hack#8] [Q&A]
Hi,
I'm trying to enable rtsp on a Tuya camera. It has the AK3918 V330L chip. I followed the instructions [here](https://github.com/MuhammedKalkan/Anyka-Camera-Firmware) to copy the `Factory` folder to the SD card. But I cannot access the camera over telnet or ftp. I then edited the `Factory/config.sh` file to add `touch /mnt/Factory/foo` to check if it was running at all, but the file was not created, so I think that file is not running at all. Any ideas? Thanks!

**Comment by rleyden559** [comment]
I spent a lot of time on AK3918 based cameras about 2 years ago and there exist several extensive GitHub threads/projects on them.  The individual models/brands are usually unlabeled and sensors and software varies without eternal indication.  I have moved on to Thingino compatible cams available is the same price range.  If your goal is too have an open device in the sub $20 price range Thingino is the way to go.
That said, I can give you my 2 year old conclusions on the AK systems I looked at.  1. I don't think rtsp can be simply enable in the stock firmware.  Custom firmware would be required.  2,  I was able to access FTP ( logging on root:root) and subsequently enabling telnet.  3. Uboot is locked even when accessing though UART and I couldn't unlock it with my skill set. 4.  Others have gone far past where I got but small variations in models and firmware made retracing their steps too frustrating for me.

**Reply by quantumfrost** [reply]
Thanks a lot for a synopsis of your efforts. Unfortunately, I wasn't even able to access ftp using root:root. I will look around a bit more, but you may be right, this is probably a lost cause. And thanks for telling me about Thingino. That's seems like the way to go.
